src/components/motor/MotorController.cpp

```diff
@@ -8,32 +8,55 @@ APP_TIMER_DEF(longVibTimer);
 
 using namespace Pinetime::Controllers;
 
+constexpr MotorController::Tune MotorController::tunes[];
+
+
 MotorController::MotorController(Controllers::Settings& settingsController) : settingsController {settingsController} {
 }
 
+
+
+uint8_t MotorController::step = 0;
+MotorController::TuneType MotorController::runningTune = MotorController::TuneType::STOP; 
+
+
+
```

Comment:
These variables only need to be declared in the header file.

Reply:
I get undefined reference to Pinetime::Controllers::MotorController::step, etc If I do not initialize it.

Reply:
Ah that's because these functions and variables are static. Usually in this project something like this is used to get around cases where a callback requires a static function.
```
void MotorController::Ring(void* p_context) {
  auto* motorController = static_cast<MotorController*>(p_context);
  motorController->RunForDuration(50);
}
```

Reply:
moved step and runningingTune from static scope. 
Would like to leave tunes[] as static. The probability is miniscule, that motorController will have multiple instances, but still.